bindings/tflite/interpreter.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IREE_BINDINGS_TFLITE_INTERPRETER_H_
+#define IREE_BINDINGS_TFLITE_INTERPRETER_H_
+
+#include "bindings/tflite/model.h"
+#include "bindings/tflite/options.h"
+#include "iree/base/api.h"
+#include "iree/hal/api.h"
+#include "iree/vm/api.h"
+
+// NOTE: we pull in our own copy here in case the tflite API changes upstream.
+#define TFL_COMPILE_LIBRARY 1
```

Comment:
Does this need to be undefed?

Reply:
nah this is all internal code (not included by user files)